Lib/Rabbit4000/BIOSLIB/StdBios.c

```diff
@@ -2613,14 +2620,15 @@ _more_inits0::
 // Initialize the status pin, clocks, clock doubler, debug baud rate
 _more_inits02::
       lcall _getDoublerSetting   ; get doubler setting value into L (h always 0)
-      ld    a, L                 ; zero value also disables /OEx early output
+      ld    a, L
       or    a                    ; update the Zero flag
       jr    z, .notEarlyOutputEnable
 
       ld    a, 0x0C              ; value to set both /OE0 and /OE1 early output
-.notEarlyOutputEnable:
 ioi   ld    (MTCR), a            ; first, update /OE0, /OE1 early output enable
       ld    (MTCRShadow), a
+
+.notEarlyOutputEnable:

```

Comment:
This change leaves `MTCR` unmodified when the clock doubler isn't enabled -- allowing the RCM4110 setting from line 1881 to remain in place.